cypress/integration/MAT/WebInterface/MeasureComposer/CQLWorkspace/ShowSQLLibraryVersion.spec.js

```diff
@@ -0,0 +1,105 @@
+import * as helper from '../../../../../support/helpers'
+import * as measurelibrary from '../../../../../pom/MAT/WI/MeasureLibrary'
+import * as measureComposer from '../../../../../pom/MAT/WI/MeasureComposer'
+import * as oktaLogin from '../../../../../support/oktaLogin'
+import * as dataCreation from '../../../../../support/MAT/MeasureAndCQLLibraryCreation'
+import * as gridRowActions from '../../../../../support/MAT/GridRowActions'
+import * as cqlComposer from '../../../../../pom/MAT/WI/CQLComposer'
+import { equal, strictEqual } from 'assert'
+import { shouldContainClick } from '../../../../../support/helpers'
```

Comment:
I think you are import helpers twice, I think you can call shouldContainClick with helper.